cypress/integration/sample.spec.ts

```diff
@@ -0,0 +1,49 @@
+describe('My First Test', () => {
+  beforeEach(() => {
+    cy.visit('http://127.0.0.1:8000/'); //change link when testing locally
```

Comment:
where does cypress know to read our source files of? 🤔

can you add a CI workflow to have those basic tests running on CI ?

Reply:
Currently, it doesn't read source files.
I ran the docs page locally and cypress ran tests against it.
For CI we can point it to `https://docs.renovatebot.com`, I will work on creating a workflow 🤞

Reply:
Can't use `https://docs.renovatebot.com` as we need to test updates as well. Will need to serve and use localhost addr